src/some-package/java/com/mainpackage/FileSyntaxError.java

```diff
@@ -0,0 +1,37 @@
+package com.mainpackage;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.S3Object;
+import java.io.IOException;
+import java.io.InputStream;
+import com.amazonaws.util.IOUtils;
+
+/**
+ * Even though this file contains weak crypto issues, CodeGuru Reviewer will not
```

Comment:
Does it? Looks like its just a syntax error.

Reply:
Missed this when copying comment. Will update in new revision